### ISSUE TITLE
atlv client window is black when started in windowed mode, or after changing fullscreen mode to windowed (or vice versa)

### ISSUE THREAD

**Issue by b2soft** [altmp/altv-issues#1827]
### Description of the problem

When I start altv client in fullscreen mode, everything works ok, but when I change windowed -> fullscreen, or fullscreen->windowed using Alt+Enter, or starting altv in Windowed mode, either altv windows becomes black, or last frame is rendered not responding visually.
Note, that button hover sounds are played.
Note, that if I connect to any server, and then press Alt+Enter, no problems observed.
![image](https://user-images.githubusercontent.com/6034932/235377791-31fbbe8f-ba36-4bea-8cd2-fc14851b4ded.png)
https://cdn.discordapp.com/attachments/576771706119520287/1101620080661504010/2023-04-28_23-23-03.mkv

Problem occurs only if `earlyLoad = true`
Possibly related to cef loses gpu context
cef.log
```[0504/190614.761:INFO:render_frame_host_impl.cc(11505)] RenderFrameHostImpl::MaybeGenerateCrashReport url = http://ui/index.html#/, status = 2, exit_code = 1
[0504/190614.763:ERROR:gpu_process_host.cc(975)] GPU process exited unexpectedly: exit_code=1
[0504/190614.763:WARNING:gpu_process_host.cc(1277)] The GPU process has crashed 1 time(s)
[0504/190614.764:ERROR:network_service_instance_impl.cc(474)] Network service crashed, restarting service.
[0504/190614.907:WARNING:gpu_process_host.cc(998)] Reinitialized the GPU process after a crash. The reported initialization time was 113 ms
```

### Reproduction steps

1. `earlyLoad = true`
2. Launch altv
3. Press Alt+Enter to change mode
4. Observe black client window/last frame rendered and not responding

### Expected behaviour

Altv client has to work in both Windowed and Fullscreen modes without problems changing between

### Additional context

I use three Monitor setup, main is 1440p (G-SYNC), secondary are 2xFullHD, RTX 3070
Checked with one monitor, no G-SYNC, Both Studio and Game Ready drivers
Checked at other PC.  with three monitors, no G-SYNC, RTX 3060, Win 11 22H2

### Operating system

Windows 10 Pro
Windows 11 22H2

### Version

15.0-dev102

### Reproduction tested

- [X] I confirm that I have made sure that this issue is also present on the newest dev version

**Comment by C0kkie** [comment]
@b2soft  could you enable netgraph and see if it freezes for you if you have the blackscreen?

**Comment by b2soft** [comment]
Windows is completely black with netgraph = true in config

**Comment by b2soft** [comment]
UPD: confirmed at other account, other PC, RTX 3060 Ti, Win 10.
cef.log:
```[0505/135217.110:ERROR:network_service_instance_impl.cc(474)] Network service crashed, restarting service.
[0505/135219.118:ERROR:gpu_process_host.cc(975)] GPU process exited unexpectedly: exit_code=1
[0505/135219.118:WARNING:gpu_process_host.cc(1277)] The GPU process has crashed 1 time(s)
[0505/135219.284:WARNING:gpu_process_host.cc(998)] Reinitialized the GPU process after a crash. The reported initialization time was 137 ms
[0505/135305.859:ERROR:gpu_process_host.cc(975)] GPU process exited unexpectedly: exit_code=1
[0505/135305.859:WARNING:gpu_process_host.cc(1277)] The GPU process has crashed 2 time(s)
[0505/135306.032:WARNING:gpu_process_host.cc(998)] Reinitialized the GPU process after a crash. The reported initialization time was 143 ms
[0505/135315.116:INFO:render_frame_host_impl.cc(11505)] RenderFrameHostImpl::MaybeGenerateCrashReport url = http://ui/index.html#/, status = 2, exit_code = 1073807364
[0505/135315.118:ERROR:network_service_instance_impl.cc(474)] Network service crashed, restarting service.
[0505/135315.141:ERROR:gpu_process_host.cc(975)] GPU process exited unexpectedly: exit_code=1073807364
[0505/135315.141:WARNING:gpu_process_host.cc(1277)] The GPU process has crashed 3 time(s)
[0505/135315.146:ERROR:network_service_instance_impl.cc(474)] Network service crashed, restarting service.
[0505/135315.171:ERROR:gpu_process_host.cc(975)] GPU process exited unexpectedly: exit_code=-1073741205
[0505/135315.171:WARNING:gpu_process_host.cc(1277)] The GPU process has crashed 4 time(s)
[0505/135315.178:ERROR:network_service_instance_impl.cc(474)] Network service crashed, restarting service.
[0505/135315.188:ERROR:gpu_process_host.cc(975)] GPU process exited unexpectedly: exit_code=-1073741205
[0505/135315.188:WARNING:gpu_process_host.cc(1277)] The GPU process has crashed 5 time(s)
[0505/135315.200:ERROR:network_service_instance_impl.cc(474)] Network service crashed, restarting service.
[0505/135315.204:ERROR:gpu_process_host.cc(975)] GPU process exited unexpectedly: exit_code=-1073741205
[0505/135315.204:WARNING:gpu_process_host.cc(1277)] The GPU process has crashed 6 time(s)
[0505/135315.204:FATAL:gpu_data_manager_impl_private.cc(451)] GPU process isn't usable. Goodbye.```

Note, that release version can become unresponsive if using Alt+tab and then Alt+Enter

**Comment by xenicle** [comment]
I can add that I have (rare) crash, black screen or unrelated image when alt+entering during altv loading. Mostly occur if I click something else in windows during altv precharging (during the altv launcher window and rockstar launcher loading) that way the game start in windowed mode. if I just click altv shortcut and wait without doing anything else the game start in full screen. (For information I have a multi monitor setup), right no I'm unable to reproduce it on dev

**Comment by b2soft** [comment]
@xenicle thanks for the info. I mostly alt tab while loading, but yeah, that seems affect only the windowed/fullscreen state, alt+enter breaks it for me and some friends 100%

**Comment by xxshady** [comment]
Thats how alt+enter looks like for me. It's not black screen, but everything is broken after pressing alt+enter. Tested with only one monitor connected to my pc.

https://youtu.be/mNA5c1RLtUY

**Comment by b2soft** [comment]
@xxshady It's the same for me, sometimes black, sometimes broken. Thanks for help

**Comment by b2soft** [comment]
quick update, for my pc at least, it starts ok, but Alt+Enter still kills it

**Comment by C0kkie** [comment]
@b2soft  Test on newest dev please

**Comment by b2soft** [comment]
Fixed in 15.0-dev147
